src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -144,7 +145,7 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
             What should others message you about? Add your bio and conversation starters here.
           </Text>
         ) : (
-          <Text style={styles.bioText}>{addDotAndRemoveNewline(bio || '')}</Text>
+          <TextWithEmoji text={addDotAndRemoveNewline(bio || '')} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the direct rendering of bio text with the `TextWithEmoji` component seems correct. However, it's important to ensure that the `addDotAndRemoveNewline` function doesn't interfere with the emoji detection in the `TextWithEmoji` component. If this function modifies the string in a way that could affect the format of emojis, it might cause issues.
```diff
- <TextWithEmoji text={addDotAndRemoveNewline(bio || '')} />
+ <TextWithEmoji text={bio || ''} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->